setup.cfg

```diff
@@ -40,17 +40,11 @@ install_requires =
 	waveorder==1.0.0rc0
 	pycromanager==0.19.2
 	click>=8.0.1
-	pyyaml>=5.4.1
-	tqdm>=4.61.1
-	opencv-python>=4.5.3.56 # candidate for deletion 1.0
 	natsort>=7.1.1
 	colorspacious>=1.1.2
 	pyqtgraph>=0.12.3
-	superqt>=0.2.4
```

Comment:
The visuals sliders (that we've hidden for now) use this. Fine to remove this dep for now.

Reply:
You'll just need to make sure to remove the import from main_widget.